### PR TITLE
"glowing AC" fix 1 - guess and check syntax until it randomly works

### PR DESCRIPTION
**Pull request by yrmvgh**

### CODE REVIEW

**Comment by yrmvgh** [comment]
got it in 2 tries! getting good at this. :green_heart: :grey_exclamation:

File: crawl-ref/source/player.cc [hunk 1]
```diff
@@ -8817,7 +8817,8 @@ bool temperature_tier (int which)
 
 bool temperature_effect(int which)
 {
-    switch (which)
+    if (you.species == SP_LAVA_ORC)
+    	switch (which)
```
Comment (jeremygurr) [inline crawl-ref/source/player.cc:8805]: the block under switch needs to be indented to the same level as switch like this:

switch (which)
{
    case LORC
...

File: crawl-ref/source/player.cc [hunk 2]
```diff
@@ -8845,6 +8846,8 @@ bool temperature_effect(int which)
         default:
             return false;
     }
+    else
+    return false;
```
Comment (jeremygurr) [inline crawl-ref/source/player.cc]: this return needs to be indented as well, otherwise it appears that it runs after the else, which it doesn't.

**Comment by yrmvgh** [comment]
Better?

**Comment by jeremygurr** [comment]
Better. But the cases are now too indented. Make sure you are using spaces and not tabs. Whatever editor you use, make sure to configure it to convert tabs to spaces, or many devs will have trouble with those files. And all indentations should be 4 spaces. Seems like these are unimportant things, but when you have half of your commits being styling changes, you realize very quickly the need for code style standards.

**Comment by yrmvgh** [comment]
Pretty sure this is deployable, at least to fix the bug.

**Pull request merged**